classroom.rb

```diff
@@ -0,0 +1,15 @@
+require './student'
+
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @student.push(student)
+    student.class_room = self
+  end
```

Comment:
- [ ] in order to make this work you will need to set up the attribute reader for `@students` as well